.circleci/config.yml

```diff
@@ -167,9 +167,14 @@ commands:
   deploy-3scale-eval-from-template:
     steps:
       - run:
-          name: Deploy 3scale from amp-eval template
+          name: Deploy 3scale from amp-eval template with nightly images
           command: |
             oc new-app --file pkg/3scale/amp/auto-generated-templates/amp/amp-eval.yml \
+              --param AMP_BACKEND_IMAGE=quay.io/3scale/apisonator:nightly \
+              --param AMP_ZYNC_IMAGE=quay.io/3scale/zync:nightly \
+              --param AMP_APICAST_IMAGE=quay.io/3scale/apicast:nightly \
+              --param AMP_SYSTEM_IMAGE=quay.io/3scale/porta:nightly \
+              --param MEMCACHED_IMAGE=registry.redhat.io/3scale-amp2/memcached-rhel7:3scale2.8 \
```

Comment:
Should we use 3scale2.9 here? In this case I'm not 100% sure as I remember in some places we needed an existing memcached image so the tests could be ran

Reply:
same comment https://github.com/3scale/3scale-operator/pull/409#pullrequestreview-440130632

Reply:
:+1: